bench_method_test.go

```diff
@@ -144,10 +138,11 @@ func TestBenchmarkMethodCall(t *testing.T) {
 	}
 
 	tOpts := TransportOptions{
+		CallerName:  "bar",
 		ServiceName: "foo",
 		HostPorts:   []string{s.hostPort()},
 	}
-	tp, err := getTransport(tOpts, encoding.Thrift)
+	tp, err := getTransport(tOpts, encoding.Thrift, nil)
```

Comment:
What is the difference in behaviour between `nil` here and `NoopTracer` in `bench_method.go`?